src/simparams.jl

```diff
@@ -42,10 +44,14 @@ struct SimParams
   phone_tracing_params::Union{Nothing, PhoneTracingParams}
 
   infection_modulation_function::Union{Nothing,InfectionModulation}
+  infection_travels_function::Union{Nothing,InfectionTravels}
+  travels_frequency::TimePoint
```

Comment:
I believe this can be made a part of the travel function

Reply:
ok, i will move it